packages/popup/src/pages/Home/components/CurrentDapp.js

```diff
@@ -2,45 +2,65 @@ import {useState} from 'react'
 import {useRPC} from '@fluent-wallet/use-rpc'
 import {useTranslation} from 'react-i18next'
 import {AuthorizeModal, DisconnectModal} from '../components'
+import {request} from '../../../utils'
+import useGlobalStore from '../../../stores/index.js'
 import {RPC_METHODS} from '../../../constants'
-const {GET_CURRENT_DAPP, GET_CURRENT_ACCOUNT} = RPC_METHODS
+const {GET_CURRENT_DAPP, GET_CURRENT_ACCOUNT, REQUEST_PERMISSIONS, DELETE_APP} =
+  RPC_METHODS
 
 function CurrentDapp() {
   const {t} = useTranslation()
+  const {setFatalError} = useGlobalStore()
   const [authModalShow, setAuthModalShow] = useState(false)
   const [disconnectModalShow, setDisconnectModalShow] = useState(false)
-  const {data: currentDapp} = useRPC([GET_CURRENT_DAPP], undefined, {
+  const {data} = useRPC([GET_CURRENT_DAPP], undefined, {
     fallbackData: {},
   })
+  const currentDapp = data?.app || {}
```

Comment:
There's also `data.site`, whose `eid` should be used in the `onAuth` function.

`data.app` will be undefined if it's not authed already.

Reply:
`onAuth` is used at the bottom of home page. When dapp is authed but not authed current account there will display connect another account. So if `data` is not `null` data.app will always be not undefined here.

Reply:
use `data.site` instead of `data.app.site`